contracts/external/traders/Expiry.sol

```diff
@@ -154,17 +182,21 @@ contract Expiry is
         public
         onlySolo(msg.sender)
     {
-        (
-            uint256 marketId,
-            uint32 expiryTime
-        ) = parseCallArgs(data);
+        ExpiryArg[] memory expiries = abi.decode(data, (ExpiryArg[]));
```

Comment:
Ethlint (aka solium) fails on this line, ruining the whole file https://github.com/duaraghav8/Ethlint/issues/261

Reply:
https://github.com/duaraghav8/Ethlint/issues/261#issuecomment-523601473